src/Sarif.Driver/Sdk/DefaultDriverOptions.cs

```diff
@@ -31,7 +31,7 @@ private DefaultDriverOptions()
                 @"Enabled state of rule. Valid values: Default, Disabled, Warning, Error. A rule in the 'Default' " + 
                 @"state will raise all issues as errors or warnings according to how the issue is logged.");
```

Comment:
Is this true?

Reply:
Yes, BUT! you have made me realize RuleEnabledState can now be entirely zapped in preference of FailureLevel. Splitting that out was really a good change, as two other enums have converged to it. i'm glad we gave this a second thought in the f2f

---
In reply to: [255311140](https://github.com/Microsoft/sarif-sdk/pull/1259#discussion_r255311140) [](ancestors = 255311140)